README.md

```diff
@@ -36,9 +36,10 @@ Smart, efficient cache solution for WordPress. Use DB, HDD, APC or Memcached for
 * We don’t handle support via e-mail, Twitter, GitHub issues etc.
 
 ### Contribute ###
-* Active development of this plugin is handled on GitHub.
+* Active development of this plugin is handled on GitHub. (see here: https://github.com/pluginkollektiv/cachify)
 * Pull requests for documented bugs are highly appreciated.
 * If you think you’ve found a bug (e.g. you’re experiencing unexpected behavior), please post at the [support forums](https://wordpress.org/support/plugin/cachify) first.
+* If you want to help us translate this plugin you can do so here: https://translate.wordpress.org/projects/wp-plugins/cachify
```

Comment:
Same here. Maybe something along the lines of `[…] translate this plugin, head over to [translate.wordpress.org](https://translate.wordpress.org/projects/wp-plugins/cachify).`